ocaml/idl/datamodel.ml

```diff
@@ -564,15 +564,17 @@ let _ =
     ~doc:"The specified IP address violates the VIF locking configuration." ();
   error Api_errors.pif_is_management_iface [ "PIF" ]
     ~doc:"The operation you requested cannot be performed because the specified PIF is the management interface." ();
+  error Api_errors.pif_not_present ["host"; "network"]
+    ~doc:"There is no PIF associated to the Host on the given network." ();
```

Comment:
Writing "with" instead of "to" would fix the grammar here, but I would write it as
"This host has no PIF on the given network."